### ISSUE TITLE
Fallout 1in2 won't start correctly and sometimes crashes

### ISSUE THREAD

**Issue by thePhilipJFry** [rotators/Fo1in2#234]
I am using the steam versions and followed the instructions to the letter according to github page and a video on how to install it. When I started the mod without tweaks it started fine. After tweaking some of the settings in the files it though it would crash. I even uninstalled both games and the zip file and did it over again. This time I made my tweaks first. Both times after tweaking the settings though, and the games starts up Fallout 2 and then crashes. I have no idea what I'm doing wrong and am feeling pretty frustrated and defeated. What am I doing wrong? Reddit has failed me

**Comment by wipe2238** [comment]
Sorry, but "some of the settings" is not enough to even start thinking about guessing what might be wrong.

There's whole bunch of settings for sfall, HRP, specific for this mod only, fallout itself... about a dozen of files with gods know how many settings in there. Pointing out what you changed and where ***might be*** a good start.

**Comment by NovaRain** [comment]
You didn't mention what settings you tweaked or what is the crash error message, and no further information provided for two days. If this issue is for expressing your feelings of frustration and defeat, I think it's already served its purpose.

**Comment by thePhilipJFry** [comment]
The only reason I 

> You didn't mention what settings you tweaked or what is the crash error message, and no further information provided for two days. If this issue is solely for expressing your feelings of frustration and defeat, I think it's already served its purpose.

The only reason I haven't replied since is 1) I was pretty mad and didn't want to think about it for a bit. 2) I've been very busy this whole weekend with my personal life

**Comment by thePhilipJFry** [comment]
To clarify the first error said something about the ddraw.ini not finding something or needing to be set to a different mode, I think?

I honestly can't remember. 

The following day on friday I tried again all day and this is what happened:

I managed to get the mod working at the time of writing this. Basically, what I did was uninstall everything, including the zip drive, and download 7zip. Then, I reinstalled Fallouts 1 and 2 played and tested both for about a minute before installing the zip drive. Then, after struggling with getting the Fallout 1in2 files in 7zip, I followed the instructions in a video I found on YouTube. After that, in an effort to follow the advice of several others struggling with the mod, I deleted the files for other languages in the Fallout 1in2 folder. It said there was an error I didn't screencap, but when I tried troubleshooting, Windows said there were some compatability issues. I went to the README and FAQ files and followed their advice just in case that would fix it. It then said, "ERROR initializing video mode 1280 X 720" I then decided to follow my intuition and change the graphics height and width to match the resolution it was possibly referencing, and lo and behold It worked. 

While I'm not happy, I can't play at a 4k resolution (or at least close to it) I think I'll take what I can get since I'm not very tech savvy when it comes modding
